tests/integ/sagemaker/jumpstart/model/test_jumpstart_model.py

```diff
@@ -160,12 +158,12 @@ def test_jumpstart_gated_model_neuron(setup):
 
 
 def test_jumpstart_gated_model(setup):
-
     model_id = "meta-textgeneration-llama-2-7b"
 
     model = JumpStartModel(
         model_id=model_id,
-        model_version="3.*",  # version >=3.0.0 stores artifacts in jumpstart-private-cache-* buckets
+        model_version="3.*",
+        # version >=3.0.0 stores artifacts in jumpstart-private-cache-* buckets
```

Comment:
Let's remember to get rid of commented codes.

Reply:
This is from js side. I don't think we should change it.